clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -312,8 +312,7 @@ public void onFailure(RuntimeException e) {
                 protocolType(),
                 metadata());
 
-        // create the request for the coordinator
-        log.debug("Issuing request ({}: {}) to coordinator {}", ApiKeys.JOIN_GROUP, request, this.coordinator.id());
+        log.debug("Sending JoinGroup ({}) to coordinator {}", request, this.coordinator);
```

Comment:
Initially I didn't use Node's toString() at all. I just filled in "{}:{}" with the host and port, but that got a little annoying after a couple times. But maybe it's actually better?